content/applications/marketing/events/attendees_reports.rst

```diff
@@ -0,0 +1,262 @@
+   drop-down menu. Doing so would show the user how many attendees each partner was associated with
+   for each event listed in the rows.
+
+   The user could take that analysis even further by clicking the :icon:`fa-plus-square` beside any
+   other available option to view more detailed options, via another drop-down menu, if desired.
+
+.. _attendees_reports/kanban:
+
+Kanban view
+-----------
+
+Click the :icon:`oi-view-kanban` :guilabel:`Kanban` icon in the upper-right corner to view the
+:guilabel:`Attendees` reporting metrics in a Kanban view.
+
+Unless manually removed from the :guilabel:`Search...` bar, the filter options specifically focus on
+attendee data :guilabel:`Taken` in the :guilabel:`Last 30 days`, and groups them by :guilabel:`Event
+> Status > Registration Date: Day`.
+
+With those default filters and grouping in place, each event in the database appears as its own
+Kanban stage, with each attendee represented by their own Kanban card beneath their respective
+event.
+
+.. image:: attendees_reports/default-kanban-view.png
+   :align: center
+   :alt: The default Kanban view of the Attendees reporting page in the Odoo Events app.
+
+Each attendee Kanban card shows their name, who they were booked by, the type of ticket they
+purchased (if applicable), an indicator if the ticket was :guilabel:`Sold` or :guilabel:`Free`,
+along with two clickable icon buttons: a :icon:`fa-check` :guilabel:`(checkmark)` icon and a
+:icon:`fa-undo` :guilabel:`(undo)` icon.
+
+When the :icon:`fa-check` :guilabel:`(checkmark)` icon is clicked, the attendee is manually marked
+as :guilabel:`Attended`.
+
+If the attendee was accidentally marked as :guilabel:`Attended`, click the :icon:`fa-undo`
+:guilabel:`(undo)` icon to revert that action, which makes the :icon:`fa-check`
+:guilabel:`(checkmark)` icon reappear on the attendee Kanban card.
+
+Like with other reporting pages, the usual :doc:`search, filter, and group by
+<../../essentials/search>` functionality can be utilized for more customized organization, if
+needed.
+
+.. example::
+   In Kanban view on the :guilabel:`Attendees` reporting page, a user could opt to remove all the
+   default filters and groupings from the :guilabel:`Search...` bar, and click the
+   :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to reveal the mega menu of
+   :guilabel:`Filters` and :guilabel:`Group By` options.
+
+   From that mega menu, the user could opt to select :guilabel:`Ongoing Events` from the
+   :guilabel:`Filters` column. Then, they could select :guilabel:`Status` from the :guilabel:`Group
+   By` column.
+
+   With those configurations in place, the :guilabel:`Attendees` reporting page would organize to
+   show two columns: :guilabel:`Attended` and :guilabel:`Registered`. This type of configuration
+   would be useful to see how many people are showing up (and expected to show up) for an event that
+   is currently taking place.
```

Comment:
```suggestion
   The report can be modified to show a company how many people are showing up (and expected to show up) for an event that
   is currently taking place. To display this information, make the following adjustments to the report. 
   
   In Kanban view on the :guilabel:`Attendees` reporting page, remove all the
   default filters and groupings from the :guilabel:`Search...` bar, and click the
   :icon:`fa-caret-down` :guilabel:`(down arrow)` icon to reveal the mega menu of
   :guilabel:`Filters` and :guilabel:`Group By` options.

   From that mega menu, select :guilabel:`Ongoing Events` from the
   :guilabel:`Filters` column. Then, select :guilabel:`Status` from the :guilabel:`Group
   By` column.

   With those configurations in place, the :guilabel:`Attendees` reporting page displays two columns: :guilabel:`Attended` and :guilabel:`Registered`. 
```

Same suggestion as above- to introduce the idea/what the report is showing, then a more active language to explain ow to achieve that report